### ISSUE TITLE
Show rate plot in chronology

### ISSUE THREAD

**Issue by Mashin6** [taginfo/taginfo#306]
Right now the plot in chronology shows cumulative function. It would be useful to be able to switch to rate (change per unit of time).
(Bonus would be to switch to barplot with month/year binning)

**Comment by ZeLonewolf** [comment]
Yes, this would be incredibly useful!  I currently use a Java program to
generate comparison tables like this[1] from the taginfo REST interface,
but it would be great to have an easier way.

[1] https://wiki.openstreetmap.org/wiki/User:ZeLonewolf/Analysis/Reservoir
